back/prisma/scripts/denormalize-form-sirets.ts

```diff
@@ -0,0 +1,76 @@
+import { Status } from "@prisma/client";
+import prisma from "../../src/prisma";
+import { registerUpdater, Updater } from "./helper/helper";
+
+@registerUpdater(
+  "Denormalize sirets",
+  `Populate forms sirets after schema denormalization`,
+  true
+)
+export class DenormalizeFormSiretsUpdater implements Updater {
+  async run() {
+    console.info("Starting denormalization script...");
+
+    try {
+      const listOfStatus = Object.values(Status);
+      console.info("1/3 Populating recipientsSirets...");
+
+      for (const status of listOfStatus) {
+        console.info(`Processing status = ${status} batch`);
+        await prisma.$executeRawUnsafe(`
+          UPDATE default$default."Form" AS f1
+          SET "recipientsSirets" =
+          array_remove(
+            array(SELECT f2."recipientCompanySiret"
+              FROM default$default."Form" AS f2
+              WHERE f2.id = f1."forwardedInId"
```

Comment:
`AND f2."recipientCompanySiret" IS NOT NULL` permettrait d'éviter le `array_remove` non ?